scrapyapperyio.py

```diff
@@ -55,7 +58,7 @@ def process_item(self, item, spider):
             def extract_session(response):
                 if response.status != 200:
                     raise RuntimeError("Unable to login: %s" % response.body)
-                self._session = json.loads(response.body)['sessionToken']
+                self._session = json.loads(str(response.body,encoding='utf-8'))['sessionToken']
```

Comment:
It all looks perfect - I would add a space after the ',' for pep8'ish (although the rest of the file isn't right now)